webapp/cypress/types/global.d.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2025, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+// cypress/types/global.d.ts
+import { TAppPages } from "./index";
```

Comment:
Use type keyword

Reply:
It's an old code, the config with imports + eslint will prevent this automatically ('verbatimModuleSyntax' error)